task_sdk/src/airflow/sdk/api/client.py

```diff
@@ -136,6 +137,11 @@ def finish(self, id: uuid.UUID, state: TerminalTIState, when: datetime):
         body = TITerminalStatePayload(end_date=when, state=TerminalTIState(state))
         self.client.patch(f"task-instances/{id}/state", content=body.model_dump_json())
 
+    def succeed(self, id: uuid.UUID, when: datetime, task_outlets, outlet_events):
+        """Tell the API server that this TI has to succeed."""
```

Comment:
```suggestion
        """Tell the API server that this TI has succeeded."""
```

Reply:
Right, it has succeeded! Not going to succeed.